home/tests/unit/api/test_leaderboard.py

```diff
@@ -0,0 +1,13 @@
+from django.test import Client, TestCase
+
+
+class TestLeaderboard(TestCase):
+    def test_Leaderboard(self):
```

Comment:
You can use the python library `faker` which is in the project to generate fake data to test with, so you can verify that the behavior works as intended... That is, create some fake accounts with fake devices and a fake contest, and store some fake steps- but control exactly how many steps to generate for each account, so you can predict what the result should be for the response payload and verity that the API is working.

Reply:
I believe I've addressed all of your comments except for this one, which I will work on.